Trustroots/trustroots
# Block users UI

#### Proposed Changes

* Let users block and unblock each other from profile and message thread UI

#### Testing Instructions

* Open a profile
* Block them from their profile or message thread

  In profile
  ![image](https://user-images.githubusercontent.com/87168/116817632-085e0b80-ab70-11eb-8770-d1fceeb248dc.png)
  
  In profile via mobile
   ![image](https://user-images.githubusercontent.com/87168/116817702-5f63e080-ab70-11eb-92d1-1882b53551e6.png)


 In message thread
   ![image](https://user-images.githubusercontent.com/87168/116817640-11e77380-ab70-11eb-8a0a-f556cd2286f0.png)

* You will see a banner informing that you've blocked them
    ![image](https://user-images.githubusercontent.com/87168/116817674-38a5aa00-ab70-11eb-9440-b613d98db93e.png)

* Login as the other user
* You shouldn't now see the first user anymore nor be able to send them messages:

Part of https://github.com/Trustroots/trustroots/issues/1467
